### ISSUE TITLE
Region 19: Outlines closer to target year 2000 exist for Kerguelen Islands

### ISSUE THREAD

**Issue by eberthie** [GLIMS-RGI/rgi7_scripts#40]
Over the Kerguelen Is (region 19, subantarctic Islands) RGI70 currently selects in GLIMS an "historical" inventory dating back from the 60s. There is also in the GLIMS database an updated inventory derived from Landsat images acquired in March and November 2001 much closer to the target 2000 year. Thanks for the fix!

**Comment by fmaussion** [comment]
Thanks for the hint!

I think these are the submissions 527 and 528: https://nbviewer.org/urls/cluster.klima.uni-bremen.de/~fmaussion/misc/rgi7_scripts/workflow/RGI19.ipynb?flush_cache=true#List-of-submissions 

I think it will be easy to use them - I'll have a look soon!

**Comment by eberthie** [comment]
Yes these are the two submissions based on the 2001 images.

**Comment by fmaussion** [comment]
I am very sorry I let this here linger for so long. I was doing clean up because RGI7 alpha is almost done (yay!) and I found this review. I had a look at the outlines:

The south-western part (Submission 527, blue) does look better than RGI (red), but:
- a few snowpatches seem to have been mapped? (Not a big deal)
- there are no ice divides

![image](https://user-images.githubusercontent.com/10050469/177052499-996c610f-a991-454e-8eca-4074b6f22313.png)

Without RGI overlay:
![image](https://user-images.githubusercontent.com/10050469/177052504-c5afc5b9-b595-4de8-9895-8af71887a06a.png)


The North and Western part (Submission 528, green ) also looks better than RGI (red), but:
- there is an obvious artifact at the southern boundary which seems to have escaped review in GLIMS
- same problem: no ice divides (this is a bit more of an issue)

![image](https://user-images.githubusercontent.com/10050469/177052598-aa7c25bc-6a30-4a0c-b529-d5164dbe407c.png)

No RGI overlay, north part: 

![image](https://user-images.githubusercontent.com/10050469/177052609-8e251de8-7936-4dad-8c43-2978b04475b0.png)

To summarize: although these look much better, I can't use them right away. If we can fix this quickly (like: couple of weeks) I would like to get them in, if not this will have to wait :disappointed: 

It's my fault not having seen this earlier.

**Comment by fmaussion** [comment]
Here is a link to the outlines by the way: https://cluster.klima.uni-bremen.de/~fmaussion/misc/rgi7_data/l0_tmp_data/rgi19_inventories.tar.gz

**Comment by fmaussion** [comment]
Frank agreed to submit before the end of the week (Juli 8)

**Comment by fmaussion** [comment]
IN RGI7
